packages/server/tests/acceptance/ws/subscribe.spec.ts

```diff
@@ -130,6 +130,11 @@ describe('@web-socket Acceptance Tests', async function() {
             await new Promise(resolve => setTimeout(resolve, 1000));
  
         });
+
+        it('Connection TTL is enforced, should close all connections', async function() {
+            await new Promise(resolve => setTimeout(resolve, parseInt(process.env.WS_MAX_CONNECTION_TTL) + 1000));
```

Comment:
If `WS_MAX_CONNECTION_TTL` is a new config please make sure to update the docs and files where configs are defined